lib/gen_stage.ex

```diff
@@ -1693,6 +1705,14 @@ defmodule GenStage do
           "GenStage.stream/1 expects a list of subscriptions, got: #{inspect(subscriptions)}"
   end
 
+  @doc """
+  Returns the number of buffered items for a producer.
```

Comment:
The documentation should be clear this is an estimate. It is not the precise amount of buffered items.